### PR TITLE
Geraine's Hosts Quest

### PR DESCRIPTION
**Pull request by TectonicRifts**
Added all 7 of Geraine's Hosts.
Added Mhoire Lieutenant with Vigor of Mhoire HP buff for killing all 7 hosts within 10 min and handing in the hearts to the Mhoire Soldier in the Graveyard.

http://acpedia.org/wiki/Geraine%27s_Hosts

### CODE REVIEW

File: Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Elemental/45710 Luminary Guard.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+     , (45710, 125,    0.25) /* ResistHealthDrain */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (45710,   1, 'Luminary Guard') /* Name */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (45710,   1,   33559685) /* Setup */
+     , (45710,   2,  150994945) /* MotionTable */
+     , (45710,   3,  536871066) /* SoundTable */
+     , (45710,   6,   67116522) /* PaletteBase */
```
Comment (DridsGhost) [inline Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Elemental/45710 Luminary Guard.sql:67]: Missing combat table

File: Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Crystal/45750 Acid Crystal.sql [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+     , (45750,  68,       5) /* TargetingTactic - Random, LastDamager */
+     , (45750,  81,       2) /* MaxGeneratedObjects */
+     , (45750,  82,       2) /* InitGeneratedObjects */
+     , (45750,  93,    3080) /* PhysicsState - ReportCollisions, Gravity, LightingOn */
+     , (45750, 133,       4) /* ShowableOnRadar - ShowAlways */
+     , (45750, 146, 1100000) /* XpOverride */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (45750,   1, True ) /* Stuck */;
+
```
Comment (DridsGhost) [inline Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Crystal/45750 Acid Crystal.sql:23]: These guys, all 4, move and fall down into the pit. Should it?

**Review by DridsGhost — COMMENTED** [review]
ace45710-luminaryguard combat table null
ace45750-acidcrystal, all crystals can fall off ledge

File: Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Crystal/45750 Acid Crystal.sql (hunk 2 above)
Comment (TectonicRifts) [reply Database/Patches/2012-04-Retributions/9 WeenieDefaults/Creature/Crystal/45750 Acid Crystal.sql:23]: No, I'll add edge slide. Nothing should be able to fall.

**Pull request merged**